### PR TITLE
[FIX]account_invoice_control: allow edit invoice

### PR DESCRIPTION
**Pull request by cav-adhoc**
Adds new field restrict_edit_invoice in account.move to restrict editing invoices only for users in group_restrict_edit_invoice

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-invoicing/pull/180).

File: account_invoice_control/__init__.py [hunk 1]
```diff
@@ -2,3 +2,4 @@
 # For copyright and license notices, see __manifest__.py file in module root
 # directory
 ##############################################################################
+from . import models
```
Comment (maq-adhoc) [inline account_invoice_control/__init__.py]: Falta en enter al final de archivo

File: account_invoice_control/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import account_move
```
Comment (maq-adhoc) [inline account_invoice_control/models/__init__.py]: Falta en enter al final de archivo

File: account_invoice_control/models/account_move.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    _inherit = "account.move"
+
+    restrict_edit_invoice = fields.Boolean(compute='_compute_restrict_edit_invoice')
+
+    def _compute_restrict_edit_invoice(self):
+        if self.env.user.has_group('account_invoice_control.group_restrict_edit_invoice'):
+            self.restrict_edit_invoice = True
+        else:
+            self.restrict_edit_invoice = False
```
Comment (maq-adhoc) [inline account_invoice_control/models/account_move.py]: Falta en enter al final de archivo

**Review by maq-adhoc — CHANGES_REQUESTED** [review]

**Review by maq-adhoc — APPROVED** [review]

**Comment by maq-adhoc** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@cav-adhoc @maq-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/account-invoicing#181

**Comment by fw-bot-adhoc** [comment]
@cav-adhoc @maq-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/account-invoicing#181

**Comment by fw-bot-adhoc** [comment]
@cav-adhoc @maq-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/account-invoicing#181

**Comment by fw-bot-adhoc** [comment]
@cav-adhoc @maq-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/account-invoicing#181

**Comment by fw-bot-adhoc** [comment]
@cav-adhoc @maq-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/account-invoicing#181

**Comment by fw-bot-adhoc** [comment]
@cav-adhoc @maq-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/account-invoicing#181

**Pull request closed without merging**